config/secrets.yaml

```diff
@@ -0,0 +1,20 @@
+orgs:
+  falcosecurity:
+    actions:
+      secrets:
+        - DOCKERHUB_SECRET
+        - DOCKERHUB_USER
+    repos:
+      falco:
+        actions:
+          secrets:
+            - GPG_KEY
+      libs:
+        actions:
+          secrets:
```

Comment:
All of these secrets are now present in our 1-password store. We should:
* move FEDEDP_GIST_SECRET to `POIANA_GIST_SECRET`
* add all of them